src/test/scala/uk/gov/hmrc/test/ui/pages/CommonElements.scala

```diff
@@ -207,6 +207,10 @@ trait CommonElements extends BasePage with Matchers{
     "state benefit change amount" -> By.cssSelector("div.govuk-summary-list__row:nth-child(4) > dd:nth-child(3) > a:nth-child(1) > span:nth-child(1)"),
     "state benefit change tax taken question" -> By.cssSelector("div.govuk-summary-list__row:nth-child(5) > dd:nth-child(3) > a:nth-child(1) > span:nth-child(1)"),
     "state benefit save and continue" -> By.cssSelector("#save-and-continue-button-id"),
+    "state benefit continue" -> By.id("continue-button-id"),
```

Comment:
We already have a continue, so I update the application to use "continue" and this key can me deleted. Use the existing one.